sky/serve/replica_managers.py

```diff
@@ -1,6 +1,8 @@
 """ReplicaManager: handles the creation and deletion of endpoint replicas."""
 import dataclasses
 import enum
+from enum import auto
+from enum import Enum
```

Comment:
nit: import module instead of functions/classes. Ref: https://google.github.io/styleguide/pyguide.html#224-decision